bootstrap/bootstrap.ps1

```diff
@@ -45,8 +46,8 @@ $rootCliVersion = Join-Path $RepositoryRoot ".cliversion"
 $dotNetCliVersion = Get-Content $rootCliVersion
 
 # now execute the script
-Write-Host "$initCliLocalPath -Version $dotNetCliVersion -InstallDir $CliLocalPath"
-Invoke-Expression "$initCliLocalPath -Version $dotNetCliVersion -InstallDir $CliLocalPath"
+Write-Host "$initCliLocalPath -Version $dotNetCliVersion -InstallDir $CliLocalPath -Architecture ""$Architecture"""
+Invoke-Expression "$initCliLocalPath -Version $dotNetCliVersion -InstallDir $CliLocalPath -Architecture ""$Architecture"""
```

Comment:
Could `initCliLocalPath` be renamed to `dotnetInstallPath` or the other way around in this PR? Want to keep the ps1 and sh as similar as possible I think. (Not what this PR is about, but same line.)